network-api/networkapi/buyersguide/migrations/0032_uplift_app_device_fields.py

```diff
@@ -0,0 +1,133 @@
+# Generated by Django 2.2.5 on 2019-11-06 20:11
```

Comment:
similar to migration 0028, which did this for a whole bunch of other Boolean fields.